java/server/src/org/openqa/selenium/grid/commands/sessionmaps.txt

```diff
@@ -25,3 +25,32 @@ After the sessionmap server successfully starts, you should see some log lines a
 15:22:55.136 INFO [KqueueProvider.<clinit>] - Starting with kqueue library
 15:22:55.328 INFO [SessionMapServer.execute] - Started Selenium session map 4.0.0-alpha-7 (revision a5536dfb7b*): http://192.168.0.5:5556
 ```
+2. JdbcBackedSessionMap
+========================
+
+`JdbcBackedSessionMap` uses the JDBC API to talk to the JDBC driver and use related database for sessions map store.
+
+The table name used to store sessions is : sessions_map
+It consists of three columns of string datatype : session_ids, session_caps, session_uri
+
+Prerequisites
+1. Ensure the appropriate JDBC driver jar is downloaded on the host machine.
+2. Ensure the "sessions_map" table with varchar "session_ids", "session_caps" and "session_uri" columns is created in the database.
+
```

Comment:
We should probably add a `CREATE TABLE` statement here to help users set up quicker.

Reply:
Sure. Added that bit.